agasc/supplement/utils.py

```diff
@@ -84,22 +87,25 @@ def get_supplement_table(name, agasc_dir=None, as_dict=False):
         except tables.NoSuchNodeError:
             warnings.warn(f"No dataset '{name}' in {supplement_file},"
                           " returning empty table")
-            dat = []
+            dat = np.array([], dtype=dtypes[name])
```

Comment:
Setting `dtypes=[]` seems weird to me. I would not be 100% sure what that does, except for guessing it is an empty structured array. I think the code is going to do the same thing if you just set it to `None` since it is just testing for a 0-length array.